docs/API/V1/daterange.md

```diff
@@ -5,4 +5,54 @@ authenticated client, optionally filtered by owner and/or access policy.
 
 For example, this can be used to initialize a date picker.
 
-__TBD__
+## Query parameters
+
+`access`    string \
+Select whether only `private` or only `public` access datasets will be included
+in the list. By default, all datasets readable by the authenticated user are
+included. For example, without constraints `/datasets/daterange` for an
+authenticated user will include all `public` datasets plus all datasets owned
+by the authenticated user; specifying `private` will show only the authenticated
+user's private datasets, while specifying `public` will show only `public`
+datasets (regardless of ownership).
+
+`owner` string \
+Select only datasets owned by the specified username. Unless the username
+matches the authenticated user, only "public" datasets can be selected.
```

Comment:
Why is this approach sufficient without the addition of `mine`?  Conversely, why wasn't this approach sufficient elsewhere where we added `mine`?  (I.e., isn't this selection criteria supposed to match or reflect the results of the actual list when we fetch it...in which case, shouldn't both calls be made using the same selection options, and therefore both calls should _offer_ the same selection options?)

Reply:
Arguably, it's not sufficient, and we should add `mine` here; but I wanted to avoid changing *code* in this documentation API. (Though I already fudged that a bit by changing dashboard code.) Really, when I added `mine` to `GET /datasets` I just didn't think of also adding it here. It's easy enough.

Reply:
> I wanted to avoid changing code in this documentation API

That's fair.  So, just open a Jira issue and we can close this conversation.  (Although, there is a small question of whether the documentation should reflect the code _as it is_ or _as it "should" be_; if it's the former, then we need to fix the documentation when we fix the code; if it's the latter, then we run the risk of falling into the skew.)